crates/frontend/src/components/override_form.rs

```diff
@@ -40,6 +48,32 @@ where
         });
     });
 
+    let get_default_config_val = move |config_key_value: String, value: String| {
+        get_config_value(
+            &config_key_value,
+            &value,
+            &default_config
+                .get_value()
+                .into_iter()
+                .map(ConfigType::DefaultConfig)
+                .collect::<Vec<_>>(),
+        )
+        .expect("can't parse default config key")
```

Comment:
Can we use unwrap_or instead and pass a json value having config_key_value : 
`unwrap_or(json!(config_key_value)`